File: Mainframe.BuyerSupplier.Ui/admin-app/src/app/delivery-slots/DeliverySlotsDto.ts
import { Time } from "@angular/common";

export class DeliverySlotsDto{
    id:number;
    slotName:string;
    firstWaveTime:string;
    secondWaveTime:string;
    startTime:string;
    cutoffTime:string;
    countdownTime:string;
    orderAcceptTime:string;
    orderCofirmTime:string;
    dispatchesConfirmTime:string;
    endTime:string;
    disabled:boolean;
}

File: Mainframe.BuyerSupplier.Core/Dto/DeliverySlotsDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Mainframe.BuyerSupplier.Core.Dto
{
    public class DeliverySlotsDto
    {
        public int ID { get; set; }

        public string SlotName { get; set; }

        public string FirstWaveTime { get; set; }

        public string SecondWaveTime { get; set; }

        public string StartTime { get; set; }

        public string CutoffTime { get; set; }

        public string CountdownTime { get; set; }

        public string OrderAcceptTime { get; set; }

        public string OrderCofirmTime { get; set; }

        public string DispatchesConfirmTime { get; set; }

        public string EndTime { get; set; }
    }
}
